### PR TITLE
Reorganize Operations info

### PR DESCRIPTION
**Pull request by dbreseman**

### CODE REVIEW

**Comment by katarzyna-koltun-mx** [comment]
Verified with Nidhi that the information was sufficient for Private Cloud and did not need updating on that end.

**Comment by katarzyna-koltun-mx** [comment]
@dbreseman I've done the changes and asked for SME feedback.

**Comment by dbreseman** [comment]
Thanks @katarzyna-koltun-mx! Looks good to me.

File: content/en/docs/developerportal/operate/_index.md [hunk 1]
```diff
@@ -14,22 +14,24 @@ When you have created your app, it is important to be able to manage it. There a
 
 Access to operations information is divided into four pages: Metrics, Alerts, Logs, and Backups.
 
+{{% alert color="info" %}}
+This page describes the operations pages for apps deployed to Mendix Cloud. If your app is deployed to SAP Business Technology Platform (SAP BTP) instead, the **Metrics**, **Alerts**, **Logs**, and **Backups** pages in the Developer Portal link to the corresponding pages on SAP BTP.
```
Comment (gauravmendix) [inline content/en/docs/developerportal/operate/_index.md]: Just an FYI , Actually all link redirect to SAP BTP. We do not point to exact page.

Comment (katarzyna-koltun-mx) [reply content/en/docs/developerportal/operate/_index.md]: @gauravmendix Thanks for the heads up - fixed in all the files.

**Pull request merged**